sui_types/src/committee.rs

```diff
@@ -68,4 +70,26 @@ impl Committee {
         // then (N + 2) / 3 = f + 1 + k/3 = f + 1
         (self.total_votes + 2) / 3
     }
+
+    /// Given a sequence of (AuthorityName, value) for values that are ordered, provide the
```

Comment:
```suggestion
    /// Given a sequence of (AuthorityName, value) for values, provide the
```

Reply:
The inputs do not need to be ordered -- we are ordering them internally (let me check we do).